src/lib/ui/editor/index.svelte

```diff
@@ -115,7 +125,11 @@
 
 	const debouncedUpdates = createDebouncedCallback(async ({ editor }) => {
 		const json = editor.getJSON();
-		content.set(json);
+
+		if (!disableLocalStorage) {
```

Comment:
We don't need to get JSON if we're not setting anything. We can do an early return at the start of the callback:
`if (disableLocalStorage) return;`

Reply:
@TGlide Don't we want to call `onDebouncedUpdate(editor)` on line 133?

does the function have a connection with localStorage?

Reply:
@rotimi-best we probably do! But it's a separate point from what I mentioned

Reply:
Oh I get what you mean. Will make the update.

Reply:
@TGlide Going back to the code, I remember what I meant. If I added an early return, line 143 won't be called. 

https://github.com/TGlide/novel-svelte/blob/eae6dcdec335fc17ef86cd64b7115732022d863f/src/lib/ui/editor/index.svelte#L136-L144

Won't that be a problem, that a prop that should be called when there is a chang isn't called when `disableLocalStorage` is `true`?

Reply:
True! You can move the getJson line inside the if then.